awx/ui_next/src/components/PromptDetail/PromptDetail.jsx

```diff
@@ -19,6 +20,19 @@ const PromptHeader = styled.h2`
   margin: var(--pf-global--spacer--lg) 0;
 `;
 
+function formatTimeout(timeout) {
+  if (typeof timeout === 'undefined' || timeout === null) {
+    return null;
+  }
+  const minutes = Math.floor(timeout / 60);
+  const seconds = timeout - Math.floor(timeout / 60) * 60;
+  return (
+    <>
+      {minutes} <Trans>min</Trans> {seconds} <Trans>sec</Trans>
```

Comment:
Is it possible to wrap <Trans> around this entire thing?  That way whoever translates this can rearrange the pieces as needed:

```
<Trans>
    {minutes} min {seconds} sec
</Trans>
```